snc.sh

```diff
@@ -49,14 +57,6 @@ OC=./openshift-clients/linux/oc
 
 run_preflight_checks
```

Comment:
same could be said about the preflight checks, maybe the pull secret check should be moved there, and the preflights moved before oc downloading?

Reply:
Sure I can try to do that, and yes pref light should run before.